### PR TITLE
nav to my basket from recipe details, use new views

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
* **#11** <a href="https://app.graphite.dev/github/pr/miamtech/MarmitonDemoIOSMealz/11?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @diarmuidmcg and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: MarmitonDemoIOSMealz/fr.lproj/LaunchScreen.strings [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (walerowicz-miam) [inline MarmitonDemoIOSMealz/fr.lproj/LaunchScreen.strings:1]: fichier inutile non ?

File: MarmitonDemoIOSMealz/fr.lproj/Main.strings [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+/* Class = "UITabBarItem"; title = "MyBasket"; ObjectID = "FNE-Eg-t0G"; */
+"FNE-Eg-t0G.title" = "MyBasket";
+
+/* Class = "UITabBarItem"; title = "Item"; ObjectID = "Fdo-rS-od3"; */
+"Fdo-rS-od3.title" = "Item";
+
+/* Class = "UITabBarItem"; title = "Details"; ObjectID = "t3X-gN-jy4"; */
+"t3X-gN-jy4.title" = "Details";
```
Comment (walerowicz-miam) [inline MarmitonDemoIOSMealz/fr.lproj/Main.strings:9]: fichier inutilise aussi il me semble

File: MarmitonDemoIOSMealz/fr.lproj/LaunchScreen.strings (hunk 1 above)
Comment (diarmuidmcg) [reply MarmitonDemoIOSMealz/fr.lproj/LaunchScreen.strings]: removed

File: MarmitonDemoIOSMealz/fr.lproj/Main.strings (hunk 2 above)
Comment (diarmuidmcg) [reply MarmitonDemoIOSMealz/fr.lproj/Main.strings]: rempved

**Pull request merged**